src/web/data/userMapData.json

```diff
@@ -1999,6 +2336,7 @@
       "University of Massachusetts",
       "University of Massachusetts Amherst",
       "University of Massachusetts Boston",
+      "University of Massachusetts, Dartmouth",
```

Comment:
Delete UMass Dartmouth, then. These two are the same.

Reply:
Same for UMass Boston

Reply:
removed